dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/StaticLinkageChecker.java

```diff
@@ -163,7 +141,8 @@ private static void printStaticLinkageReport(StaticLinkageCheckReport report) {
   /**
    * Finds linkage errors in the input classpath and generates a static linkage check report.
    */
-  StaticLinkageCheckReport findLinkageErrors() throws ClassNotFoundException, IOException {
+  // TODO why does this throw ClassNotFoundException? Shouldn't that just be part of the report?
```

Comment:
Maybe file an issue for this one?

Reply:
https://github.com/GoogleCloudPlatform/cloud-opensource-java/issues/263

Reply:
I thought `ClassNotFoundException` represented an unexpected condition or bug in the checker, not something that is an expected result of a report.